docs/examples/bootstrap.php

```diff
@@ -52,33 +46,6 @@
 // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 echo "Using test root $_tests_dir\n";
 
-// WordPress requires PHPUnit 7.5 or earlier and hacks around a few things to
-// make it work with PHP 8. Unfortunately for MockObjects they do it via
-// composer.json rather than bootstrap.php, so we have to manually do it here.
-if ( version_compare( PHP_VERSION, '8.0', '>=' ) &&
-	( ! class_exists( PHPUnit\Runner\Version::class ) || version_compare( PHPUnit\Runner\Version::id(), '9.3', '<' ) )
-) {
-	if ( ! class_exists( PHPUnit\Framework\MockObject\InvocationMocker::class, false ) &&
-		file_exists( "$_tests_dir/includes/phpunit7/MockObject/InvocationMocker.php" )
-	) {
-		// phpcs:disable WordPressVIPMinimum.Files.IncludingFile.NotAbsolutePath
-		require "$_tests_dir/includes/phpunit7/MockObject/Builder/NamespaceMatch.php";
-		require "$_tests_dir/includes/phpunit7/MockObject/Builder/ParametersMatch.php";
-		require "$_tests_dir/includes/phpunit7/MockObject/InvocationMocker.php";
-		require "$_tests_dir/includes/phpunit7/MockObject/MockMethod.php";
-		// phpcs:enable
-	} else {
-		fprintf(
-			STDOUT,
-			"Warning: PHPUnit <9.3 is not compatible with PHP 8.0+, and the hack could not be loaded.\n  Class %s exists: %s\n  File %s exists: %s\n",
-			PHPUnit\Framework\MockObject\InvocationMocker::class,
-			class_exists( PHPUnit\Framework\MockObject\InvocationMocker::class, false ) ? 'yes (bad)' : 'no (good)',
-			"$_tests_dir/includes/phpunit7/MockObject/InvocationMocker.php",
-			file_exists( "$_tests_dir/includes/phpunit7/MockObject/InvocationMocker.php" ) ? 'yes (good)' : 'no (bad)'
-		);
-	}
-}
-
 // Give access to tests_add_filter() function.
```

Comment:
Let's copy in the check for `composer install` having been run that was added to Jetpack in #20392
```suggestion
if ( ! is_readable( $_plugin_root . '/vendor/autoload.php' ) ) {
	echo 'The plugin is not ready for testing.' . PHP_EOL;
	echo PHP_EOL;
	echo 'Composer dependencies must be installed.' . PHP_EOL;
	exit( 1 );
}

// Give access to tests_add_filter() function.
```